docs/monitoring/10-01-prometheus-istio-server-oom.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Prometheus Istio Server restarting or in crashback loop due to OOM
+type: Troubleshooting
+---
+
+Prometheus Istio Server scrapes metrics from all the envoy side car. It might crash because of OOM when the cardinality of the istio metrics is increasing too much. This is usually is happening for a high amount of workloads performing a lot of communication to other workloads. This issue can be tackled in following ways:
```

Comment:
```suggestion
Prometheus Istio Server scrapes metrics from all envoy side cars. It might crash because of OOM when the cardinality of the Istio metrics increases too much. This usually happens when a high amount of workloads perform a lot of communication to other workloads. This issue can be tackled by following these steps:
```